docs/Embedded/EmbeddedSetup.md

```diff
@@ -7,9 +7,27 @@ title: Installing Enklu Embedded
 
 Enklu Embedded enables Unity Developers to embed Enklu Cloud content within Unity scenes and sync 3D Assets to Enklu Cloud directly from Unity. With Enklu Embedded, Unity Developers enjoy the freedom of Unity and the rapid iteration and real-time synchronization of Enklu Cloud. Enklu Embedded works for all Unity projects, but is specifically optimized to improve workflows for immersive augmented reality developers.
 
-## Installing
+Enklu Embedded is distributed as a [Unity Package](https://docs.unity3d.com/Manual/PackagesList.html) and can be added to your Unity project with our [installer](https://cdn.enklu.com/web-assets/packages/EnkluEmbeddedInstaller.unitypackage) or with Unity's Package Manager. 
 
-Enklu Embedded is distributed as a [Unity Package](https://docs.unity3d.com/Manual/PackagesList.html) and can be installed with Unity's Package Manager. To register the Enklu package repository with the Package Manager, add an entry to the `scopedRegistries` section of the package manifest in your project. 
+> If your project uses the Unity package available on our guide [Exporting Unity Prefabs](/docs/Assets/ExportingUnityPrefabs), remove it before installing the Enklu Embedded packages.
+
+## Using the Installer
+
+To make getting started as seamless as possible, we have created an installer that automatically imports the Enklu Embedded package and all its dependencies:
+
+[`EnkluEmbeddedInstaller.unitypackage`](https://cdn.enklu.com/web-assets/packages/EnkluEmbeddedInstaller.unitypackage)
+
+To run the installer, simply drag it into your Unity project or import it from the `Asset > Import Package > Custom Package...` menu and confirm the two files to import.
+
+![Importing the Installer](/img/product/embedded/Installer.png)
+
+Once the files have been added to the project, the installer will import the Enklu Embedded package, if necessary. Please note that the process can take several minutes. It will perform this check every time you launch your Unity project. It is also possible to run the `Enklu > Install Enklu Embedded...` menu command. 
```

Comment:
Also, maybe this is more of my ignorance on the subject, but what exactly is performing the check every time I launch my Unity project? Is it the package manager? Or is it the installer?

Reply:
will clarify